### ISSUE TITLE
The sample code in the readme makes Sendgrid send back a 400.

### ISSUE THREAD

**Issue by BillBrower** [elbuo8/sendgrid-django#64]
When I format the `to` address as directed in the readme I get a 400 back from Sendgrid.

```python
mail = EmailMultiAlternatives(
  subject="Your Subject",
  body="This is a simple text email body.",
  from_email="Yamil Asusta <hello@yamilasusta.com>",
  to=["yamil@sendgrid.com"],
  headers={"Reply-To": "support@sendgrid.com"}
)
```

```json
{
	"errors": [{
		"message": "Invalid type. Expected: string, given: array.",
		"field": "personalizations.0.to.0.email",
		"help": "http://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/errors.html#message.personalizations.to"
	}]
}
```

**Comment by jayhale** [comment]
This is fixed by #58. Recommend closing.

Evidence: https://github.com/jayhale/sendgrid-django-64

**Comment by BillBrower** [comment]
Thanks @jayhale!
